.github/workflows/test.yml

```diff
@@ -0,0 +1,41 @@
+name: tests 🔬
+
+on:
+  push:
+    branches:
+      - 'master'
```

Comment:
Probably not for now, but would be nice to switch to `main` as the default branch

Reply:
Yeah I have that in my todo, but didn't want to get too overwhelmed with this. I plan to do it after this merges.